default.nix

```diff
@@ -0,0 +1,11 @@
+(
+  import
+    (
+      builtins.fetchTarball {
+        url = "https://github.com/edolstra/flake-compat/archive/12c64ca55c1014cdc1b16ed5a804aa8576601ff2.tar.gz";
+        sha256 = "sha256-yDCjQCE2EEr1GnqJyVNb6orbcTS1G7A9IXshNk/1uvs=";
+      }
+    )
+    {
+      src = ./.;
+    }).defaultNix
```

Comment:
This could also be an option. I've added the `nix-compat` input to the flake already. So we won't have to update the hashes ourselves every time.
```nix
(import (let lock = builtins.fromJSON (builtins.readFile ./flake.lock);
in fetchTarball {
  url =
    "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
  sha256 = lock.nodes.flake-compat.locked.narHash;
}) { src = ./.; }).defaultNix
```

Reply:
Oh, even better. I wasn't sure whether to go that route at first.